### PR TITLE
fix: Fixing Overlapping Components in `Manage Events`

### LINKED ISSUE #3240: Ember Table: Overlapping Components of Ember Table in `Manage Events`
Various Components are overlapping in `Manage Events` Section

### PR DESCRIPTION
**Pull request by kushthedude**
Fixes #3240 

#### Short description of what this resolves:

- A workaround for fixing `Overlapping Components` in `Ember Table`

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/extensions/ember-table/template.hbs [hunk 1]
```diff
@@ -1,5 +1,5 @@
-<div class="resize-container">
-  <table class="ui unstackable table">
+<div class="ui segment column sixteen wide inner-table-wrapper x-scrollable">
+  <table class="ui very basic unstackable table">
```
Comment (CosmicCoder96) [inline app/extensions/ember-table/template.hbs]: we don't want a basic table.

File: app/extensions/ember-table/template.hbs [hunk 2]
```diff
@@ -1,5 +1,5 @@
-<div class="resize-container">
-  <table class="ui unstackable table">
+<div class="ui segment column sixteen wide inner-table-wrapper x-scrollable">
```
Comment (CosmicCoder96) [inline app/extensions/ember-table/template.hbs]: inner-table-wrapper is not doing anything here.

**Comment by kushthedude** [comment]
@CosmicCoder96 @niranjan94 Just tried to tweak with column widths and its best combination I found which prevents collapsing in smallest mobile view too. Till we don't have any good solution, I think it can be merged.

**Comment by kushthedude** [comment]
@CosmicCoder96 @niranjan94 Please check.

**Pull request merged**